src/Services/privateApiService.js

```diff
@@ -21,4 +21,16 @@ const Get = () => {
     .catch((err) => console.log(err));
 };
 
-export default Get;
+export const privatePATCH = async (path, id, body) => {
+  try {
+    const response = await axios.patch(`${path}/${id}`, body, config);
+
+    console.log(response.data);
```

Comment:
Please remove the console.log() as they should not be on production code. (The below console.error() is fine)